src/containers/Clusters/Clusters.tsx

```diff
@@ -44,6 +48,15 @@ export function Clusters() {
 
     const dispatch = useTypedDispatch();
 
+    React.useEffect(() => {
+        dispatch(setHeaderBreadcrumbs('clusters', {}));
```

Comment:
Is it possible to set it as default value in store instead of useEffect here?

Reply:
1. It is not a default value, there is no clusters page in single-cluster version
2. `useEffect` is needed here anyway to reset current breadcrumbs state. So even if it will be a default value, some function call will be needed